pkgs/build-support/docker/default.nix

```diff
@@ -761,16 +792,27 @@ rec {
   mergeImages = images: runCommand "merge-docker-images"
```

Comment:
This function has fixed arguments and cannot easily be extended to take the new `compressor` argument. To work around this, I simply made it pick the compression schema used in the last image.